### PR TITLE
add psql && ssh2 ext for php image

### PR DESCRIPTION
**Pull request by fabienChaillou**

### CODE REVIEW

File: php/Dockerfile.tpl [hunk 1]
```diff
@@ -82,4 +82,12 @@ zend_extension=opcache.so \n\
     apk del --purge alpine-sdk autoconf && \
     rm -rf /tmp/* /usr/share/doc /var/cache/apk/* && \
 
+    echo "Starting ssh2!" && \
```
Comment (mremi) [inline php/Dockerfile.tpl]: Please move it above with other extensions installation

Comment (mremi) [reply php/Dockerfile.tpl]: Please also update `CHANGELOG.md` and `test.php`

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ Versions
 * Add a Chrome image (based on Alpine Node, with NPM, Yarn and Puppeteer)
 * Fix pip version to 18 and pipenv to 2018.7.1 to avoid recent conflict between these tools
 * Add dependencies for cpython libs in dind-aws image
+* Add `memcached`, `pgsql`and `ssh2` extension in PHP images
```
Comment (mremi) [inline CHANGELOG.md]: `extensions` 😉

File: php/Dockerfile.tpl [hunk 3]
```diff
@@ -36,8 +37,14 @@ RUN echo "Starting ..." && \
...
     docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ && \
     docker-php-ext-configure intl && \
-    docker-php-ext-install -j$(getconf _NPROCESSORS_ONLN) bcmath exif gd intl pcntl pdo_mysql pdo_pgsql soap sockets zip && \
+    docker-php-ext-install -j$(getconf _NPROCESSORS_ONLN) bcmath exif gd intl pcntl pdo_mysql pdo_pgsql pgsql soap sockets zip && \
     pecl install apcu-${APCU_VERSION} && \
+    pecl install memcached-${MEMCACHED_VERSION} && \
+    docker-php-ext-enable memcached && \
+    docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql && \
+    curl -o /tmp/ssh2.tgz https://pecl.php.net/get/ssh2 && \
+    pecl install ssh2 /tmp/ssh2.tgz && \
```
Comment (mremi) [inline php/Dockerfile.tpl]: `pecl install ssh2` without `curl` before does not work?

Comment (fabienChaillou) [reply php/Dockerfile.tpl]: curl is already installed (line 23).
I check if ssh2 extension exist on my local
@mremi

**Pull request merged**